website/content/docs/operations/metrics.mdx

```diff
@@ -0,0 +1,97 @@
+---
+layout: docs
+page_title: Boundary Metrics
+description: |-
+  Obtain visibility of various components of a running Boundary
+---
+
+## Boundary Metrics
+
+Boundary provides metrics through the `/metrics` path using a listener with the
+"ops" purpose. By default a listener with that purpose runs on port 9203.  See
+the [sample code](#sample-code) section for an example listener stanza in a
+`config.hcl` file.
+
+Metrics are important for understanding what is happening in a boundary
+deployment, enabling diagnoses of deployed controllers and workers. Adding
+metrics will grant greater visibility into the various components of a running
+Boundary deployment through Prometheus.
+
+For more information about Prometheus, see the [Prometheus Overview
+Guide](https://prometheus.io/docs/introduction/overview). The Prometheus Go
+library is used to define and export metrics from the Boundary system. The
+metrics uses the [OpenMetric
+exposition](https://prometheus.io/docs/instrumenting/exposition_formats/#openmetrics-text-format)
+format.
+
+
+### Controller
+
+The following lists the current names and descriptions of metrics provided for controllers.
+
+| Name                                                          | Description                                   |
+|---------------------------------------------------------------|-----------------------------------------------|
+| `boundary_controller_api_http_request_duration_seconds`       | Historgram of latencies for HTTP requests.     |  
+| `boundary_controller_api_http_request_size_bytes`             | Histogram of request sizes for HTTP requests.  |
+| `boundary_controller_api_http_response_size_bytes`            | Histogram of response sizes for HTTP requests. |
+| `boundary_controller_cluster_grpc_request_duration_seconds`   | Histogram of latencies for requests made to the gRPC service running on the cluster listener. |
+| `boundary_controller_cluster_grpc_request_size_bytes`         | Histogram of byte sizes for requests made to the gRPC service running on the cluster listener. |
+| `boundary_controller_cluster_grpc_response_size_bytes`        | Histogram of byte sizes for responses from the gRPC service running on the cluster listener. |
```

Comment:
These are no longer provided and should be removed.

Reply:
Removed ([commit](https://github.com/hashicorp/boundary/pull/1977/commits/78bde823f2d445e36869c2c7b619d7587e9675c6))